### PR TITLE
Improved User Interface

### LINKED ISSUE #30: User Interface  Improvement
**UI Improvements:**

- Currently in MainFragment when there is no profile present, it shows a png image, So, we can replace that image with some 
good Lottie Animation that will improve UI & UX.
- Currently icon at splash screen is png image (which is looking little bit blur/faddish), we can replace it with other svg image which will look clear.
- Currently color of status bar in Splash screen is black (with white texts in it), So, we can change status bar color to white with black color texts in it.

**Screenshots of current state**

Screenshot_20201010-162017_Quiet_Hours 1

Screenshot_20201010-162045_Quiet_Hours 1

**Would you like to work on the issue ?**
- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by Haid-Faiz**
Fixes #30 

Changes: Added Lottie Animation on MainFragment (it will show when there is no profiles in the list) and Changed status bar color to white and it's text color to black (On Splash Screen) and changed splash screen and app icon (added svg icon).
***Link of video: https://drive.google.com/file/d/1aZU7gO3LyIveuHYAXqVye07pUTbSZdk6/view?usp=drivesdk***

Screenshots for the change:

Screenshot_20201017-203517_Quiet_Hours 1

Screenshot_20201019-123729_Quiet_Hours 1

### CODE REVIEW

**Comment by Haid-Faiz** [comment]
@rob729 Kindly check this..

**Comment by rob729** [comment]
@Haid-Faiz I didn't ask you to change the text font on the home screen and icon on the splash screen. Revert those changes.

**Comment by Haid-Faiz** [comment]
@rob729 Ok..

**Comment by rob729** [comment]
Also, you might need to change the animation on the home screen as it is not looking good in dark mode. I will update you regarding it soon.

**Comment by Haid-Faiz** [comment]
@rob729  i revert those changes...

**Comment by Haid-Faiz** [comment]
> Also, you might need to change the animation on the home screen as it is not looking good in dark mode. I will update you regarding it soon.

@rob729 Sure

File: app/src/main/res/drawable/quiet_hour_icon.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (rob729) [inline app/src/main/res/drawable/quiet_hour_icon.xml:1]: which icon is this?

Comment (Haid-Faiz) [reply app/src/main/res/drawable/quiet_hour_icon.xml:1]: @rob729  i downloaded it from FlatIcon.com, it is the same icon which is already present in the project... but old icon was not looking sharp, So, i downloaded the same vector image icon. (It is looking sharp now)

**Comment by rob729** [comment]
@Haid-Faiz Change the Lottie animation to https://lottiefiles.com/32921-volumn-from-mute-to-maximum#.
One change is required in this animation, change the color of animation to #42A5F5.
You can change the color of animation using the "Edit Layer Color" option present on the website.

**Comment by Haid-Faiz** [comment]
@rob729 Ok

**Comment by Haid-Faiz** [comment]
@rob729  I changed the lottie animation

File: app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt [hunk 2]
```diff
@@ -105,11 +104,26 @@ class MainFragment : Fragment() {
                     .navigate(MainFragmentDirections.actionMainFragmentToNewProfileFragment())
             }
         }
-
+        currentlyActiveProfile()
         setHasOptionsMenu(true)
         return binding.root
     }
 
+    private fun currentlyActiveProfile() {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt]: @Haid-Faiz I think you have incorrectly resolved the merge conflicts. This change has nothing to do with your PR.
I would suggest reverting the last changes you made, then merge the master code and then push the code in the PR.

Comment (Haid-Faiz) [reply app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt]: @rob729  I did it like.... after making changes like adding animation, i commited these changes then i pull the code of your master branch (upstream) then it gets merged in my local master branch  then after squashing the commits.... i forced push the code to my remote (origin) master branch then i created PR.

Did i do it in wrong way...???

Comment (rob729) [reply app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt]: No. It seems to be right. Can you revert the recent commits and do this process again?
You can also try by first merging the master code, make the changes to the code, and then squash the commits.

Comment (Haid-Faiz) [reply app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt]: @rob729 okk

**Comment by Haid-Faiz** [comment]
@rob729 Now please have a look...

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @Haid-Faiz 25 points

**Pull request merged**